internal/profile/_static/elastic-agent_default.env

```diff
@@ -1,3 +1,4 @@
 FLEET_ENROLL=1
-FLEET_INSECURE=1
-FLEET_URL=http://fleet-server:8220
\ No newline at end of file
+FLEET_URL=https://fleet-server:8220
```

Comment:
Done in https://github.com/elastic/integrations/pull/3443, also checked manually, and I guess that is also covered by `test-stack-command-oldest`. I think we are fine in that sense.

The only problems I found were https://github.com/elastic/elastic-package/pull/789#issuecomment-1109971673, but got solved by installing the CA in the container instead of using configuration, as suggested by Tiago in https://github.com/elastic/elastic-package/pull/789#issuecomment-1111000736.